app/src/components/notifier/Notifier.test.js

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { MemoryRouter } from 'react-router-dom';
+import { mount } from 'enzyme';
+import fetchMock from 'fetch-mock';
+
+import configureStore from '../../store/configureStore';
+import AutoHideDuration from './Notifier';
+import App from '../app/App';
+import {HttpService} from '../../services/httpService';
+import {QUERY_LEN_ERR} from "../../utils/errors";
+
+const div = document.createElement('div');
+let store;
+beforeEach(() => {
+  store = configureStore();
+});
+
+function sleep (time) {
+  return new Promise((resolve) => setTimeout(resolve, time));
+}
+
+it('Notifier does not open snackbar for valid query', () => {
+
+  const SEARCH_STR = 'foobar';
+
+  const wrapper = mount(
+    <Provider store={store}>
+      <MemoryRouter>
+        <App/>
+      </MemoryRouter>
+    </Provider>);
+
+  //Check for class name of Notifier Snack Bar. Initially not present.
+  expect(wrapper.find('.Notifier-root-120').hostNodes().length).toEqual(0);
```

Comment:
@davemasselink  - Unfortunately, it does not work without the -120.

Reply:
darn, too bad... probably makes sense then to add some commentary around the line indicating that the number is somewhat arbitrary but seems to be consistent enough, for now, to base tests on. But that it would be a good thing to check back on these tests if they suddenly start to fail and it's not otherwise obvious as to why